### ISSUE TITLE
Header navigation button not read out as "collapsed"

### ISSUE THREAD

**Issue by domoscargin** [alphagov/govuk-frontend#1932]
In the [Header component with navigation](https://design-system.service.gov.uk/components/header/with-service-name-and-navigation/index.html), the first time the "Menu" button receives focus, it is not read out as "collapsed" using VoiceOver on MacOS.

## Steps to replicate
* With VoiceOver on:
* Zoom in or resize screen until nav menu collapses to the Menu button.
* Keyboard tab to the menu button - button is not labelled as "collapsed"
* Use button - button expands and is correctly labelled as "expanded"
* Use button again - button collapses and is correctly labelled as "collapsed".
